app/controllers/concerns/helpeable.rb

```diff
@@ -6,6 +6,11 @@ module Helpeable
 
   included do
 
+    def validate_uuid string
```

Comment:
so this is because in the UPSERT the clients are providing the ID, but we want all the IDs to look the same. OTherwise they can add randoms ids, such as "My-report-feb-2018-patito-123-test".  The validation will return an error if they do not send a uuid as their ID

Reply:
Ah I didn't realise they could submit random ID's okay then, makes sense